packages/shared/auth0-client/src/index.ts

```diff
@@ -181,6 +222,26 @@ export default class Auth0Client {
     });
   }
 
+  private getInstanceWithCredentials(username: string, password: string): Promise<AxiosInstance> {
+    return axios.post(this.apiRoot + '/oauth/token', {
+      client_id: this.clientId,
+      client_secret: this.clientSecret,
+      audience: this.apiAudience,
+      grant_type: 'password',
+      username,
```

Comment:
I assume that if we don't provide a key here, it'll just use the name of the variable...?

Reply:
Yeah, `{ username }` is syntax sugar for `{ username: username }`